webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelectSelectors.js

```diff
@@ -0,0 +1,15 @@
+import Immutable from 'seamless-immutable';
+
+export const mapSelected = selected => selected.map(item => item.label || item);
+
+const selectTypeAheadSelect = ({ typeAheadSelect }, id) =>
+  typeAheadSelect[id] || {};
+
+export const selectOptions = (state, id) => {
```

Comment:
Since you may import selectors from several areas of the store in a single file, prefer a more specific selector name like `selectTypeaheadOptions`.

Reply:
Don't we have `import foo as bar` to solve that? I see what you're saying but naming things in that way adds a certain amount of redundancy